### PR TITLE
[16.01] Fix misplaced parenthesis

### PR DESCRIPTION
**Pull request by nsoranzo**
Reported by @dpryan79 in https://github.com/galaxyproject/tools-iuc/issues/906#issuecomment-238844304

### CODE REVIEW

File: lib/tool_shed/galaxy_install/tool_dependencies/recipe/tag_handler.py [hunk 1]
```diff
@@ -418,9 +418,9 @@ def handle_complex_repository_dependency_for_package( self, elem, package_name,
                                                                                                         dependent_install_dir,
                                                                                                         tool_dependency_type='package' )
                     if not can_install_tool_dependency:
-                        log.debug( "Tool dependency %s version %s cannot be installed (it was probably previously installed), " %
-                            ( str( tool_dependency.name, str( tool_dependency.version ) ) ) )
-                        log.debug( "so appending it to the list of handled tool dependencies." )
+                        log.debug( "Tool dependency %s version %s cannot be installed (it was probably previously installed), "
+                                   "so appending it to the list of handled tool dependencies.",
+                                   str( tool_dependency.name), str( tool_dependency.version ) )
```
Comment (mvdbeek) [inline lib/tool_shed/galaxy_install/tool_dependencies/recipe/tag_handler.py:423]: you're missing  `%`

Comment (dannon) [inline lib/tool_shed/galaxy_install/tool_dependencies/recipe/tag_handler.py:423]: Not necessary using the `log.<level>` functions, string params are passed as extra arguments.

**Comment by nsoranzo** [comment]
I've added a commit to pin flake8 and ignore `.tox/` and `eggs/` directories, should fix py26-lint and py27-lint tests.

File: lib/tool_shed/galaxy_install/tool_dependencies/recipe/tag_handler.py (hunk 1 above)
Comment (mvdbeek) [inline lib/tool_shed/galaxy_install/tool_dependencies/recipe/tag_handler.py:423]: Good to know, thanks!

**Comment by martenson** [comment]
thanks @nsoranzo !

**Comment by nsoranzo** [comment]
Thanks @martenson, I'm forward merging now.

**Pull request merged**